homework6.py

```diff
@@ -0,0 +1,47 @@
+# 1. Створити телефонну книгу, яка матиме наступні команди:
+# stats: кількість записів
+# add: додати запис
+# delete <name>: видалити запис за іменем (ключем)
+# list: список всіх імен в книзі
+# show <name>: детальна інформація по імені
+# Записи не мають повторюватися, заборонено перезаписувати записи, тільки видаляти і додавати заново.
+
+my_contacts = {
+    'Wednesday Addams': '+442255667788',
+    'Gregory House': '+112233445566',
+    'Winston Churchill': '+445566778899'
+}
+
```

Comment:
Add `while True` here to have an ability to run not only one command but use it as a running program

Reply:
As I understood correctly I have created infinite loop to make a running program

Reply:
That is correct